backend-dashboard-2/app/controllers/api/v1/users_controller.rb

```diff
@@ -48,6 +51,6 @@ def set_user
 
     # Only allow a list of trusted parameters through.
     def user_params
-      params.fetch(:user).permit(:id, :email)
+      params.fetch(:user).permit(:email, :employee_id)
```

Comment:
Why did params for `user_params` change?

Reply:
I changed the class so it could receive the employee_id when created.
Like this:
```Ruby
field :id
  field :employee_id
  field :email, type: String
  belongs_to :employees, class_name: "Employee", foreign_key: "employee_id"
```
But it still has their inate user_id as you use it in the auth. But it's not necessary in the user_params as it's only referenced when updating the user from a PUT method.
```Ruby
 # PATCH/PUT /users/1
  def update
    if @user.update(user_params)
      render json: @user
    else
      render json: @user.errors, status: :unprocessable_entity
    end
  end
```
It can still be modified and accessed but from the back, since the only one who should touch that id, is the back-end when modyfing the current session id.

Reply:
Gotchu, thx. 👍

Reply:
Y la pelea de raaaaap